pkg/occlient/occlient.go

```diff
@@ -3277,6 +3277,29 @@ func (c *Client) IsRouteSupported() (bool, error) {
 	return false, nil
 }
 
+// IsImageStreamSupported checks if imagestream resource type is present on the cluster
+func (c *Client) IsImageStreamSupported() (bool, error) {
+
+	const ClusterVersionGroup = "image.openshift.io"
```

Comment:
can we clean this up by extracting `ClusterVersionGroup`,`Version` and `resourceName` as function parameters and the generalise it for everything?
Maybe even refactor `IsRouteSupported` to use the extracted function?